mainapp/templates/mainapp/request_form.html

```diff
@@ -38,8 +38,8 @@
       </div>
       <div id="map"></div>
       <div class="alert alert-info">
-        Use the marker to update the location. Touch / click at the bottom of the pin.
-        സ്ഥലം അപ്ഡേറ്റുചെയ്യാൻ മാർക്കർ ഉപയോഗിക്കുക. പിൻയുടെ താഴെയായി സ്പർശിക്കുക / ക്ലിക്കുചെയ്യുക.
+        Use the marker to update the location. Touch / click at the bottom of the pin and drag. Update GPS coordinate field by clicking 'Update location'.<br/>
```

Comment:
bottom of the pin and drag. Afterwards update Address field by clicking 'Update location'